learning_assistant/views.py

```diff
@@ -76,6 +79,15 @@ def post(self, request, course_run_id):
         unit_id = request.query_params.get('unit_id')
 
         message_list = request.data
+
+        # Check that the last message in the list corresponds to a user
+        new_user_message = message_list[-1]
```

Comment:
I don't really understand the code block and the comment. Can you explain more to help someone like to understand the logic better?

Reply:
What if the message_list array is empty?

Reply:
I assumed the endpoint was to process a new user message. Maybe I misunderstood it but if it is, then it should include at least one user message.

Reply:
Just to point out: We discussed this offline and came to the conclusion that the approach is correct.